src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -58,6 +58,11 @@ public ReadOnlyPerson getPerson() {
     }
 
     @Override
+    public boolean isMutating() {
```

Comment:
Header comment is required for overriden methods to inform other what the overriding is for. Same for below.